### PR TITLE
Misc wording fixes, while learning how to test/contribute

### PR DESCRIPTION
**Pull request by mblomdahl**
## Description ✏️ 
- Remove duplicate discrimination topic in Code of Conduct
- Correct capitalization of some brands (PeerTube, GitHub)
- Replace `npm` with `yarn` in Contributing guide, for additional consistency

### CODE REVIEW

**Comment by mblomdahl** [comment]
> Thanks! But we still use `npm` in dev workflow to run peertube scripts.

Thanks for clarifying, I thought they were equivalent. Updated/reverted with 5020ab1! ✔️

File: .github/CONTRIBUTING.md [hunk 1]
```diff
@@ -125,8 +125,8 @@ $ sudo -u postgres psql -c "CREATE EXTENSION pg_trgm;" peertube_dev
 $ sudo -u postgres psql -c "CREATE EXTENSION unaccent;" peertube_dev
 ```
 
-Peertube also requires a running redis server, no special setup is needed for
-this.
+PeerTube also requires a running redis server, no special setup is needed for
+this (it's `sudo service redis-server start` on WSL 1 with Ubuntu).
```
Comment (Chocobozzz) [inline .github/CONTRIBUTING.md:129]: Please remove the `(it's `sudo service redis-server start` on WSL 1 with Ubuntu)` as it's not the proper place to have detailed explanations depending on the linux distribution

Comment (mblomdahl) [inline .github/CONTRIBUTING.md:129]:
```suggestion
this.
```

Thank you @Chocobozzz! I will try to write something good over at [`./support/doc/dependencies.md`](https://github.com/Chocobozzz/PeerTube/blob/develop/support/doc/dependencies.md) documentation instead when I have everything working well under Windows/WSL. Removing this remark!

**Comment by Chocobozzz** [comment]
Thanks!

**Pull request merged**